### DISCUSSION TITLE
The new version not works : Google Chrome

### DISCUSSION THREAD

**Discussion by masterrox111112** [aclap-dev/video-downloadhelper#582] [Help and Q&A]
Whatsa happen?

version: 9.0.1.13

Would a downgrade be possible? This version was poorly made, it doesn't work.
Send me the downgrade link.
Discard this new upgrade. 
Please test it first...
Not once, but 100x

---
In any case, thank you very much for the great work. 
It went wrong once 
10000x
;D

**Comment by paulrouget** [comment]
Which version of Chrome you're using?

**Reply by artymarty15** [reply]
Different user here, but I would love a downgrade option to be available if possible. I originally had issues with videos not downloading at all, but it now seems a mix between that and downloads going incredibly slow in comparison to before the upgrade.
I was very happy with the previous version and the previous UI too. However the new one has lost the option to right click on the video and select download versions that way. That may be something to do with Chrome, but the only thing I updated was the VDH Chrome extension.
Here's my version of Chrome:
Version 124.0.6367.208 (Official Build) (64-bit)

It'll be a real huge help for many if this is something that can be done, as I've seen many people mention this being a problem.
Thank you,
Kind regards,
Marty

**Reply by paulrouget** [reply]
It would help if you could describe more precisely what stopped working exactly? Which URLs used to work and then stopped working.

You can revert to the old UI like this: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface

**Reply by artymarty15** [reply]
It isn't a problem to one specific URL, but to any that I try. They all seem to be fairly slow in comparison to what they used to be, even when reverting to the old UI. It doesn't seem to be that, but something else.
It may be something else that has changed that has caused this. I just wanted to let you know, so you know for the future versions and hopefully you'll be able to provide a way to revert back to the previous version.
If that's not possible, hopefully you can take these comments from everyone and revert changes and work from the previous version to fix things. It seems to be the same thing for many people and as I said, it's not one set URL, but anything we try. I'm regularly getting the download erroring out, straight up cancelling itself out immediately or a download just either hangs or takes hours to download. Something that would originally take usually 30-40 mins at max.
Thank you

**Reply by paulrouget** [reply]
Did you have to update your CoApp? What version of the CoApp you're using?

**Reply by BarbzYHOOL** [reply]
the right click on a page no longer show a video download for me on firefox, but now i discovered I can click on the extension button and see a menu to download, didn't even know for 2 years........

**Comment by masterrox111112** [comment]
The problem is not related to the browser or version.
And yes, the update you carried out.
OK?
Thanks

Note: There are several complaints
When it happens to one, it's one thing. With tens and hundreds is another....

**Comment by artymarty15** [comment]
Yeah, I updated both to the new version when it had prompted the update. I don't tend to use that though, but yes that got updated as well.

Martyn
________________________________

**Comment by coloradomountainguy** [comment]
I echo many of the other sentiments here.  Is there a way we could revert to the old version?  There are just too many miscellaneous bugs with the new version.  It's disappointing they didn't give us an option to go with the new version and/or do proper testing.  I'd like to be able to revert back to the old version which worked very well.

**Comment by HasBing** [comment]
I am having the exact same problem. On firefox and chrome both. The new update broke the app for me. I tried to use the old UI, but this did not help. I also tried to make the old UI settings always use co-app. This did not either. Please offer a downgrade option or fix the issue. Thank you.
